dbt/models/iasworld/schema/iasworld.pardat.yml

```diff
@@ -259,6 +259,14 @@ sources:
             description: '{{ doc("shared_column_year") }}'
           - name: tieback
             description: '{{ doc("shared_column_tieback_key_pin") }}'
+            tests:
+              - expression_is_true: 
+                  name: iasworld_pardat_tieback_does_not_contain_hyphen
+                  expression: |
+                    not like '%-%'
+                  additional_select_columns: 
+                    - parid
+                    - taxyr
```

Comment:
**[Nitpick]** We can use a [YAML alias](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/) here to select the same set of columns as the majority of the other tests in this file. Let's also add a `config` attribute and set it to the `*unique-conditions` alias so that we can apply the same filtering by date and active records that the other tests apply:

```suggestion
                  additional_select_columns: *select-columns
                  config: *unique-conditions
```

For reference, here are the definitions of the `&select-columns` and `&unique-conditions` anchors, so you can see what they're doing:

https://github.com/ccao-data/data-architecture/blob/14095e2a9bb15106ae95807f2e8302a34a3b7e45/dbt/models/iasworld/schema/iasworld.pardat.yml#L82-L86

https://github.com/ccao-data/data-architecture/blob/14095e2a9bb15106ae95807f2e8302a34a3b7e45/dbt/models/iasworld/schema/iasworld.pardat.yml#L106-L110